### PR TITLE
feat(info-section): add info-section layout to home page

### PR DESCRIPTION
**Pull request by Wizardgreen**
## Types of changes

* **New feature**

## Description

Create Info-section for Home page.

### CODE REVIEW

File: components/core/layout/TwoColWrapper.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="two-col-layout">
         <div class="left-col">
-            <slot></slot>
+            <slot name="left-col"></slot>
```
Comment (mattwang44) [inline components/core/layout/TwoColWrapper.vue:4]: This is not relevant to the PR and do break the other page (`2022/<lang>/registration/financial-aid`). Please revert the change.

File: pages/index.vue [hunk 2]
```diff
@@ -48,7 +48,46 @@
         </div>
 
         <i18n-page-wrapper>
-            <div class="sponsor-section">
+            <div class="info-section section-wrapper">
```
Comment (mattwang44) [inline pages/index.vue:51]: The mobile layout does not conform to the [design spec](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=31%3A1821). Please check again.
image

**Review by mattwang44 — CHANGES_REQUESTED** [review]
Since we implemented [a component](https://github.com/pycontw/pycontw-frontend/tree/main/components/intro) for this section last year, if you want to do it from scratch (though not recommended), please delete the old version.

File: components/intro/Intro.vue [hunk 3]
```diff
@@ -62,50 +58,28 @@ export default {
...
 }
 
 .info-section-description {
-    @apply pl-14;
-    @media (max-width: 767px) {
-        @apply pl-0 my-0 mx-auto;
-    }
-}
-
-.intro-milestones-title {
```
Comment (mattwang44) [inline components/intro/Intro.vue]: the title should emerge both on web view and mobile view (there is a small error on figma).

File: components/intro/Intro.i18n.js [hunk 4]
```diff
@@ -22,16 +22,23 @@ export default genI18nMessages({
         pyconIntro: 'PyCon Taiwan',
         whatIsPyConAPAC: 'What is PyCon APAC',
         whatIsPyConAPACContent:
-            ' PyCon APAC is the annual gathering of the global Python community. In addition to the wonderful speeches on various aspects of Python, the annual meeting also gathers people who use Python in various fields in Taiwan to exchange experiences and share the latest technological developments.',
+            'PyCon APAC is the annual gathering of the global Python community. ' +
```
Comment (mattwang44) [inline components/intro/Intro.i18n.js:25]: I'm currently asking for help to revise the translation ([discord link](https://discord.com/channels/752904426057892052/946041922827472897/972428476185448549)).

Comment (mattwang44) [reply components/intro/Intro.i18n.js]: updated
reference: https://discord.com/channels/752904426057892052/946041922827472897/972428476185448549

**Review by flynnhou — APPROVED** [review]
LGTM.

Note that we need to audit the screen breakpoints. We should be able to eliminate media queries by using tailwind's `screens` variants.

**Pull request merged**